src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -145,6 +225,13 @@ export class Accept extends Component<AcceptProps, AcceptState> {
             }
           />
         </HorizontalPadding>
+        <ErrorModal
+          onClose={this.onCloseModal}
+          show={this.state.isErrorModalOpen}
+          contactEmail="orders@artsy.net"
```

Comment:
Since Reaction is open source, should this come from an environment variable instead?

Reply:
That makes sense to me, though I'd prefer to knock that out in a separate PR since it affects a bunch of other files.

Reply:
This might fit well into our upcoming "extract common things" knowledge share/mob programming session.